bahmni_account/report/report_invoice_inherit.xml

```diff
@@ -85,12 +85,14 @@
 					 <span id="dic" t-esc="'%.2f'%(o.invoice_seach().discount)"></span><br></br>
 					 <!--<label for="net_amt"> <span style="margin-right: 74px;">Net Amount</span>:</label>
 					 <span id="net_amt" t-esc="'%.2f'%((o.invoice_seach().amount_total - o.invoice_seach().discount)+o.invoice_seach().round_off_amount)"></span><br></br>-->
+					 <label for="net_amt" class="answer"><span style="margin-right: 78px;"> Net Amount</span>:</label>
+					 <span id="net_amt" class="answer" t-esc="'%.2f'%((o.invoice_seach().amount_total - o.invoice_seach().discount)+o.invoice_seach().round_off_amount)"></span><br></br>
+					 <label for="pre_bal"><span style="margin-right: 40px;">Previous Balance</span>:</label>
+					 <span id="pre_bal" t-esc="'%.2f'%(o.invoice_seach().order_id._total_receivable() - o.invoice_seach().amount_residual)"></span><br></br>
 					 <label for="bill_amt" class="answer"><span style="margin-right: 78px;"> Bill Amount</span>:</label>
-					 <span id="bill_amt" class="answer" t-esc="'%.2f'%((o.invoice_seach().amount_total - o.invoice_seach().discount)+o.invoice_seach().round_off_amount)"></span><br></br>
+					 <span id="bill_amt" class="answer" t-esc="'%.2f'%(((o.invoice_seach().amount_total - o.invoice_seach().discount)+o.invoice_seach().round_off_amount) + (o.invoice_seach().order_id._total_receivable() - o.invoice_seach().amount_residual))"></span><br></br>					 
```

Comment:
I think there is a typo in `invoice_seach` method name. It should be `invoice_search`